### DISCUSSION TITLE
Issue installing hvplot

### DISCUSSION THREAD

**Discussion by alexiemillikin** [earthlab-education/Earth-Analytics-2023-01-Intro#173] [Q&A02 Open Reproducible Science] [answered]
Hi all, 

I am having an issue with importing "hvplot.pandas" for the coding challenge in notebook 01. I have tried the work around shown in the "Week 2 Coding Challenge: Reproducible Science" video to install the package using "mamba install -c conda-forge hvplot" in terminal, but I do not get the same output, rather my terminal reads "All requested packages already installed". My code passes the test for importing the hvplot.pandas library, but the "import hyplot.pandas" line is not the right color and I can't plot anything . Does anyone know where I'm going wrong or anything else I could check for? 

I have double-checked that I am using the base python kernel and for typos throughout. All the rest of my code for this challenge seems to work.

Thanks!

**Comment by eculler** [comment]
Hi @alexiemillikin,

Hm, that's strange. It does sound like hvplot is installed.

I notice that your wrote "import hyplot.pandas" instead of "import hvplot.pandas" in your question. Was that a typo here, or is it also in your code? If it's in the code that may fix the problem!

**Comment by alexiemillikin** [comment, marked as answer]
@eculler oops--that was just a typo in my comment, but not in the actual code. However, the code seems to be working today and now the graph shows up!

**Reply by eculler** [reply]
Sweet! Glad it’s working - let us know if it pops up again.
